docs/src/plotfunctions.md

```diff
@@ -1,6 +1,6 @@
 # Plot Functions
 
-Before using the plot recipes of `Testparticle.jl`, you need to install the package [`TestParticleMakie.jl`](https://github.com/henry2004y/TestParticle.jl/tree/master/TestParticleMakie) and import it via `using TestParticleMakie`. All plot recipes depend on `Makie.jl`.
+Before using the plot recipes of `Testparticle.jl`, you need to import `Makie` package via `using Makie`. All plot recipes depend on `Makie.jl`.

```

Comment:
I think `using GLMakie` or `using CairoMakie` also implicitly import `Makie` itself, so we don't necessarily need to `using Makie`.